modules/openapi-generator/src/main/resources/tiny-cpp-client/helpers-header.mustache

```diff
@@ -0,0 +1,24 @@
+#ifndef _HELPERS_H_
+#define _HELPERS_H_
+
+#include <string>
+#include "bourne/json.hpp"
+
+using namespace std;
```

Comment:
This should be removed. Stuff from std namespace will leak everywhere where helpers are used. If you really need it, then please move it to headers-body, but I'd prefer prefixing with `std::` everywhere it's required.

https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice